react/src/App.js

```diff
@@ -28,7 +27,7 @@ function App() {
           break;
         case "signIn_failure":
         case "cognitoHostedUI_failure":
-          console.log("Sign in failure", data);
+          console.error("Sign in failure", data);
```

Comment:
pls remove console logging. and don't think we use hosted UI anyway.

Reply:
Removed on last commit